README.md

```diff
@@ -35,28 +41,75 @@ Before you can run the bot yourself, you'll need to set up an application on Dis
 
 ### Configuring
 
-High Quality Bot expects a file called `config.cfg` to be in the same directory as the .jar file, and won't run without it. The format is simply one `key=value` declaration per line. Currently available options to set are:
+High Quality Bot expects a file called `config.cfg` to be in the same directory as the .jar file, and won't run without
+it. The format is simply one `key=value` declaration per line. `exampleconfig.cfg` provides a sample configuration.
+Currently available options to set are:
 * `token` - Your Bot account's OAuth token
 * `prefix` - The desired character (or string of characters if you're a crazy person) used to invoke commands.
+* `ownerid` - The User attached to this ID is automatically given permission to use all commands.
+* `allowed` - This is a list of command names separated by commas that users are allowed to use by default.
+* `disallowed` - Users are by default explicitly disallowed to use these commands.
+* `allowedHasPrecedence` - If true, any commands in both `allowed` and `disallowed` are considered allowed.
+* `permissiongroups` - A list of filenames separated by commas from which HQBot will read permission groups.
+At the bare minimum you should set the token and ownerid. *If the owner ID is not set, you will not be able to run any
+commands.* For more information about the Permissions system, see below.
 
 Example `config.cfg`:
 
 ```
 # Parser will ignore lines starting with '#'
+# Options are set with:
+# key=value
 
 # Token (this one isn't valid don't even try it)
-token=MjY4MjExODI2MzU1ODYzNTUz.DByjTQ.1ycw5crEGVZGo-ckyGakydsno0o (this isn't a valid token don't even try it)
+token=MjY4MjExODI2MzU1ODYzNTUz.DByjTQ.1ycw5crEGVZGo-ckyGakydsno0o
 
 # Prefix
 prefix=.
 
+# OwnerID
+ownerid=123456789
+
+# Allowed & Disallowed
+allowed=*
+disallowed=shutdown
+
+allowedHasPrecedence=false
+
 ```
 
-From here High Quality Bot should be ready to run. Place the .jar and config in the same directory wherever you like, and run it with `java -jar highqualitybot-<version>.jar`
+## Permissions
+
+High Quality Bot makes use of a permissions system that integrates with Discord's Roles. When checking if a user is
+allowed to run a command, the bot starts with the permissions set in the `config.cfg`. From there, another .cfg file is
+added for each Discord Role you would like the bot to make use of. This file includes lists of commands that users in
+the role will be granted or denied permission to use, as well as a priority in the event of collision between two roles.
+Once the file is created, simply add its path to `permissiongroups` in the main config.
+
+Example Permission Group `DJ.cfg`
+
+```
+# Lines starting with '#' are ignored
+
+# RoleID
+role=123456789
+
+# Allowed
+allowed=play,pause,skip,queue
+
+# No Disallowed commands required
+
+# Priority (should be set higher than any conflicting groups you don't want to override this one)
```

Comment:
> groups `that|which` you don't want to override this one

I misread this as a run-on sentence so we may be able to make this a little more clear